conductor/resources/resources.yml

```diff
@@ -162,6 +162,43 @@ maya_dependency_attrs:
         - filename
     PxrVisualizer: # Renderman
         - matCap
+    RedshiftBokeh:
+        - dofBokehImage
+    RedshiftCameraMap:
+        - tex0
+    RedshiftDomeLight:
+        - tex0
+        - tex1
+    RedshiftEnvironment:
+        - tex0
+        - tex1
+        - tex2
+        - tex3
+        - tex4
+    RedshiftIESLight:
+        - profile
+    RedshiftLensDistortion:
+        - LDimage
+    RedshiftLightGobo:
+        - tex0
+    RedshiftNormalMap:
+        - tex0
+    RedshiftOptions:
+        - irradianceCacheFilename
+        - irradiancePointCloudFilename
+        - photonFilename
+        - subsurfaceScatteringFilename
+    RedshiftPostEffects:
+        - clrMgmtOcioFilename
+        - lutFilename
+    RedshiftProxyMesh:
+        - computedFileNamePattern
```

Comment:
I could be wrong - (and it may not apply to Redshift anyway) but I think you don't need both filename and computedFilename. I'm sure it doesn't hurt to have both though.

Reply:
good call.  It looks like the `computedFileNamePattern` attr always returns what we want (no need for the `fileName` attr ).  I've also removed the `fileName` attr from the `RedshiftVolumeShape` node as well.